modules/label-namespaces.adoc

```diff
@@ -10,6 +10,14 @@ The {product-title} Operator uses an opt-in webhook mechanism to ensure that pol
 
 You must label the namespaces where you want {product-title} to manage jobs with the `kueue.openshift.io/managed=true` label.
 
+.Prerequisites
+
+Ensure that you have completed the following prerequisites:
```

Comment:
Is this intro sentence a new practice, perhaps DITA-related? I don't think it is a very common construct to have an intro sentence before prereqs.